test/utils.go

```diff
@@ -19,14 +18,42 @@ import (
 	"github.com/pkg/errors"
 )
 
+const fixtureBasePath = "../fixtures/"
```

Comment:
this code is not related with the fix for issue #246, please consider openning another PR

Reply:
Done